Quiet/Quiet/Screens/Detail/NoiseBarChartView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  NoiseBarChartView.swift
+//  Quiet
+//
+//  Created by Minkyeong Ko on 2022/08/25.
+//
+
+import UIKit
+
+import Charts
+
+final class NoiseBarChartView: UIView {
+    private var horizontalBarChart: HorizontalBarChartView!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        drawBarChart()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func drawBarChart() {
+        horizontalBarChart = HorizontalBarChartView()
+        addSubview(horizontalBarChart)
+        
+        horizontalBarChart.translatesAutoresizingMaskIntoConstraints = false
+        self.horizontalBarChart.constraint(
+            top: self.topAnchor,
+            leading: self.leadingAnchor,
+            bottom: self.bottomAnchor,
+            trailing: self.trailingAnchor
+        )
+    }
+    
+    func setupBarChart(dataPoints: [String], values: [Double]) {
+        var dataEntries: [BarChartDataEntry] = []
+        for i in 0..<dataPoints.count {
```

Comment:
p5;

```suggestion
        for i in dataPoints.indices {
```
이렇게도 쓸 수 있을 거 같아요!👀